Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -84,6 +84,25 @@
         }
     }
 
+    &--tfoot-border {
+        tfoot {
+            tr {
+                th {
+                    border-top: $table__tfoot-th-border-top\@medium;
+                    border-bottom: 0;
+
+                    &[scope="row"] {
+                        border-right-width: 1px;
+                    }
+                }
+                td {
```

Comment:
please add empty line. it should always appear between selectors on the same level